packages/sources/crypto-api-v2/src/config.ts

```diff
@@ -0,0 +1,12 @@
+import { Requester } from '@chainlink/ea-bootstrap'
+import { Config } from '@chainlink/types'
+
+export const DEFAULT_ENDPOINT = 'price'
+export const NAME = "CRYPTO_API_V2"
+
+export const makeConfig = (prefix?: string): Config => {
+  const config = Requester.getDefaultConfig(prefix, true)
```

Comment:
Just noticed that we are making the API key required here, but we're not marking it as required in the README.

Reply:
We should have the API_KEY env var in the readme regardless, to indicate that this adapter can use it.

Reply:
should we include the timeout as well then?  It's being used when setting up the default config.

Reply:
I don't think so, because API_TIMEOUT is global and applied to every single adapter. API_KEY is only used on a subset of adapters, and a subset of these have them as required. The README should make it clear if this is supported and/or required.